Gulpfile.mjs

```diff
@@ -480,10 +488,15 @@ const libBundles = [
   "packages/babel-plugin-bugfix-safari-id-destructuring-collision-in-function-expression",
 ].map(src => ({
   src,
-  format: "cjs",
+  format: USE_ESM ? "esm" : "cjs",
   dest: "lib",
 }));
 
+const cjsBundles = [
+  // This is used by Prettier, @babel/register and @babel/eslint-parser
+  { src: "packages/babel-parser" },
```

Comment:
We(Prettier) don't need cjs, our code base has migrate to esm, and we can do both static and dynamic import in parser, we support async parser now.

Reply:
Thanks, I'll update the comment!